app.js

```diff
@@ -15,5 +16,18 @@ app.get('/solve', function(request, response){
   response.sendFile('html/solve.html', { root: __dirname });
 });
 
+//This is for the home page
```

Comment:
I think these route handlers are small and obvious enough by themselves.
Maybe consider removing the comments as it doesn't tell anything new?

A more informative and less verbose comment would probably be a simple note such as
`// Individual route handlers` at the top of all the route handlers